code/modules/hydroponics/seeds.dm

```diff
@@ -233,7 +233,10 @@
 			if(istype(T, /obj/item/reagent_containers/food/snacks/grown))
 				var/obj/item/reagent_containers/food/snacks/grown/grown_edible = T
 				data = grown_edible.tastes
-
+				for(var/i in genes)
+					if(istype(i, /datum/plant_gene/trait/brewing) && grown_edible.distill_reagent)
+						T.reagents.add_reagent(grown_edible.distill_reagent, amount)

```

Comment:
just so you're aware, the normal way this is done is to replace the nutriment (and I think one other specific chem? I forget) in the plant with the distilled reagent. this'd let you control the amount of antihol that barrelmelons, for example, would produce.